packages/utils/test/schema/sanitizeDataForNewSchemaTest.ts

```diff
@@ -258,6 +258,37 @@ export default function sanitizeDataForNewSchemaTest(testValidator: TestValidato
         })
       ).toEqual({});
     });
+    it('returns empty formData after resolving schema refs', () => {
```

Comment:
I'm curious about this test. What prompted you to write it?

Reply:
For some reason the two changes to `resolveAnyOrOneOfSchemas` and `getClosestMatchingOption` caused lose of coverage in `sanitizeDataForNewSchema`. Maybe because the refs were being resolved and it not longer needed to in `sanitizeDataForNewSchema`?